docs/modules/plugins/pages/plugin-browser-stack.adoc

```diff
@@ -45,7 +45,7 @@ TIP: Links to BrowserStack test runs can be found in Allure report
 
 == Properties
 
-* See xref:tests-development:tests-configuration.adoc#_selenium_grid[Selenium Grid] properties to configure generic options.
+* See xref:ROOT:tests-configuration.adoc#_selenium_grid[Selenium Grid] properties to configure generic options.
```

Comment:
I think we should refactor the location of Selenium Grid configuration, it's not a common one, it's applicable only for `vividus-plugin-web-app`, `vividus-plugin-mobile-app` and test clouds plugins

Reply:
Isn't it out of the scope of the pull request? :)